maintainers/maintainer-list.nix

```diff
@@ -1365,6 +1365,12 @@
     githubId = 5684605;
     name = "Cole Scott";
   };
+  contrun = {
```

Comment:
can you make a separate commit out of the maintainers addition?

Reply:
Done